templates/webapi/test/overview.html.ep

```diff
@@ -128,15 +128,15 @@
                 <div class="row" id="filter-modules">
                     <div class="col-4">
                         <div class="form-group">
-                            <label for="filter-modules"><strong>Module</strong></label>
+                            <label for="filter-modules"><strong>Module by name</strong></label>
```

Comment:
This is not a grouping but just a filtering criteria. So I wouldn't use "by" here:
```suggestion
                            <label for="filter-modules"><strong>Module name</strong></label>
```